doc/build-unix.md

```diff
@@ -2,195 +2,166 @@ UNIX BUILD NOTES
 ====================
 Some notes on how to build Gridcoin in Unix.
 
-(For BSD specific instructions, see build-*bsd.md in this directory.)
-
-Note
----------------------
-Always use absolute paths to configure and compile Gridcoin and the dependencies,
-for example, when specifying the path of the dependency:
-
-	../dist/configure --enable-cxx --disable-shared --with-pic --prefix=$BDB_PREFIX
-
-Here BDB_PREFIX must be an absolute path - it is defined using $(pwd) which ensures
-the usage of the absolute path.
+(For BSD specific instructions, see build-\*bsd.md in this directory.)
 
 Preparing the Build
---------------------
+-------------------
 
 Install git:
-Ubuntu & Debian: `sudo apt-get install git`
-openSUSE: `sudo zypper install git`
+* Ubuntu & Debian: `sudo apt install git`
+* openSUSE: `sudo zypper install git`
+
 Clone the repository and cd into it:
 
 ```bash
 git clone https://github.com/gridcoin-community/Gridcoin-Research
 cd Gridcoin-Research
 git checkout master
 ```
+
 Go to platform specific instructions for the required dependencies below.
 
 To Build
----------------------
+--------
 
-```bash
-./autogen.sh
-./configure
-make # use "-j N" for N parallel jobs
-make install # optional
-```
+* With CMake:
 
-Or, to keep the source directory clean:
-```bash
-./autogen.sh
-mkdir build
-cd build
-../configure
-make # use "-j N" for N parallel jobs
-```
-
-This will build gridcoinresearch (Qt client) as well if the dependencies are met.
-
-Dependencies
----------------------
+  ```bash
+  mkdir build && cd build
+  cmake ..
+  cmake --build .  # use "-j N" for N parallel jobs
+  cmake --install .  # optional
+  ```
```

Comment:
Shouldn't you include the instructions to do a full build including Qt, as many people want both? i.e.

cmake .. -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DENABLE_GUI=ON -DENABLE_QRENCODE=ON -DENABLE_UPNP=ON -DUSE_DBUS=ON

is the closest analog to a full autotools build, using ccache.

Reply:
added